functional-tests/test-bed/src/main/java/org/arquillian/smart/testing/ftest/testbed/project/ProjectBuilder.java

```diff
@@ -20,7 +20,8 @@
 
 public class ProjectBuilder {
 
-    private static final String TEST_REPORT_PREFIX = "TEST-";
+    public static final String TEST_REPORT_PREFIX = "TEST-";
+    public static final String TEMP_REPORT_DIR = ".reports";
```

Comment:
The variables are used even in the `ProjectBuilder` and `TestReportHandler`  is in test scope at the moment. 
Do you then suggest moving `TestReportHandler` to the main ?